client/components/jetpack/daily-backup-status/status-card/backup-last-failed.tsx

```diff
@@ -0,0 +1,45 @@
+import { Gridicon } from '@automattic/components';
+import { ExternalLink } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { recordTracksEvent } from 'calypso/state/analytics/actions/record';
+import type { Moment } from 'moment';
+
+type Props = {
+	baseBackupDate: Moment;
+	eventsCount: number;
+	selectedBackupDate: Moment;
+	learnMoreUrl?: string;
+};
+
+export const BackupLastFailed: FunctionComponent< Props > = () => {
+	const translate = useTranslate();
+	const learnMoreLink = 'https://jetpack.com/support/backup/troubleshooting-jetpack-backup/';
+
+	return (
+		<div className="backup-last-failed__wrapper">
+			<span className="backup-last-failed__info">
+				<Gridicon icon="notice-outline" />
+				{ translate(
+					"We encountered some issues with the latest backup, but don't worry! " +
+						"We're using the most recent backup available. " +
+						'{{ExternalLink}}Learn more.{{/ExternalLink}}',
+					{
+						components: {
+							ExternalLink: (
+								<ExternalLink
+									href={ learnMoreLink }
+									children={ null }
+									rel="noopener noreferrer"
```

Comment:
I think the rel attribute is [not needed for this component](https://github.com/WordPress/gutenberg/blob/f50e045c1ce75acd026ab760891cd5bf57234fd3/packages/components/src/external-link/index.tsx#L27C8-L27C20).

Reply:
Removed. Thanks!